Gtk-4.0.gir

```diff
@@ -40988,6 +41041,12 @@ with either API.</doc>
           </parameter>
         </parameters>
       </method>
+      <property name="allowed-apis" writable="1" transfer-ownership="none" setter="set_allowed_apis" getter="get_allowed_apis" default-value="GDK_GL_API_GL | GDK_GL_API_GLES">
```

Comment:
Misses a version

Reply:
It does have the annotation upstream though... https://gitlab.gnome.org/GNOME/gtk/-/blob/main/gtk/gtkglarea.c#L891-920 not sure what is wrong here

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5824

Reply:
The new version does not include the fix yet. We need to wait another day or so, I guess.

Reply:
Looks good now